--- ./pytorch/CMakeLists.txt	2024-10-10 18:19:50.622837929 -0400
+++ ./pytorch_1/CMakeLists.txt	2024-10-10 18:07:46.396352738 -0400
@@ -1144,10 +1144,11 @@
   if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
     string(APPEND CMAKE_C_FLAGS " --coverage -fprofile-abs-path")
     string(APPEND CMAKE_CXX_FLAGS " --coverage -fprofile-abs-path")
-  elseif("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
-    string(APPEND CMAKE_C_FLAGS " -fprofile-instr-generate -fcoverage-mapping")
-    string(APPEND CMAKE_CXX_FLAGS
-           " -fprofile-instr-generate -fcoverage-mapping")
+    elseif("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+    # string(APPEND CMAKE_C_FLAGS " -fprofile-instr-generate -fcoverage-mapping")
+    # string(APPEND CMAKE_CXX_FLAGS
+    #        " -fprofile-instr-generate -fcoverage-mapping")
+    SET(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fprofile-instr-generate -fcoverage-mapping")
   else()
     message(
       ERROR
